pages/write_comment.php

```diff
@@ -0,0 +1,26 @@
+<?php
+    require "../blocks/connect_to_db.php";
+
+    $result = $mysql->query("SELECT * FROM `comments`");
+    
+    if($_COOKIE['user_id'] === NULL){
+        header('Location: /');
+        exit();
+    }
+
+    $text = filter_var(trim($_POST['textarea__comment']),FILTER_SANITIZE_STRING);
+
+    $id_of_user = $_COOKIE['user_id'];
+
+    $id_of_article = $_GET["id"];
+
+
+    if(mb_strlen($text) == 0 || mb_strlen($text) >= 255){
+        echo "<h1>Введите комментарий корректно</h1>";
```

Comment:
Change message so that it corresponds to validation logic. Specify min and max boundaries in error message